airflow/providers/google/cloud/operators/gcs_to_bigquery.py

```diff
@@ -298,10 +298,15 @@ def execute(self, context):
                 cluster_fields=self.cluster_fields,
                 encryption_configuration=self.encryption_configuration)
 
+        if bq_hook.use_legacy_sql:
+            escaped_table_name = '[{}]'.format(self.destination_project_dataset_table)
+        else:
+            escaped_table_name = '`{}`'.format(self.destination_project_dataset_table)
```

Comment:
Should we use f-strings?

Reply:
I didn't know about f-strings before, but sure!